fixtures/blocks/src/Post.js

```diff
@@ -11,10 +11,7 @@ import {fetch, preload} from 'react-data/fetch';
 import loadComments from './Comments';
 
 function load(params) {
-  if (params.id < 3) {
-    const nextId = params.id + 1;
-    preload('http://localhost:3001/posts/' + nextId);
-  }
+  preload('http://localhost:3001/comments?postId=' + nextId);
```

Comment:
Changed this because it didn't make sense. Preload should be used for things in current request, not next pages. This wouldn't work with the server model.